pom.xml

```diff
@@ -327,20 +305,44 @@
 	</build>
 
 	<profiles>
+		<profile>
+			<id>default</id>
+			<activation>
+				<property>
+					<name>!release</name>
+				</property>
+			</activation>
+			<modules>
+				<module>docs</module>
+				<module>spring-cloud-gcp-dependencies</module>
+				<module>spring-cloud-gcp-pubsub</module>
+				<module>spring-cloud-gcp-core</module>
+				<module>spring-cloud-gcp-cloudfoundry</module>
+				<module>spring-cloud-gcp-storage</module>
+				<module>spring-cloud-gcp-starters</module>
+				<module>spring-cloud-gcp-samples</module>
+				<module>spring-cloud-gcp-autoconfigure</module>
+				<module>spring-cloud-gcp-pubsub-stream-binder</module>
+				<module>spring-cloud-gcp-data-spanner</module>
+				<module>spring-cloud-gcp-data-datastore</module>
+				<module>spring-cloud-gcp-security-iap</module>
+				<module>spring-cloud-gcp-vision</module>
+				<module>spring-cloud-gcp-logging</module>
+				<module>spring-cloud-gcp-data-firestore</module>
+				<module>spring-cloud-gcp-bigquery</module>
+				<module>spring-cloud-gcp-security-firebase</module>
+				<module>spring-cloud-gcp-secretmanager</module>
+			</modules>
+		</profile>
+
 		<profile>
 			<id>spring</id>
+			<activation>
```

Comment:
@meltsufin - Do you remember why a profile was needed for the repositories? I think the Spring team once told us that our root pom should not included repositories as they will be inherited by users. Can't quite recall though.

Reply:
That's a good point. `!excludeSpring` would mean the staging repos are activated by default.

Reply:
Reverted